examples/azure/function_app_powershell/function_app_powershell.tf

```
@@ -0,0 +1,31 @@
+# --- RESOURCES / MODULES ---
+
+module "function_app_powershell" {
+  # source = "github.com/THEY-Consulting/they-terraform//azure/function-app"
+  source = "../../../azure/function-app"
+
+  name                = "they-test-powershell"
```

QoL: Would prefer dynamic naming to ensure we have no problems when multiple people in the same account deploy an example (assuming everyone works in their own workspace) and to avoid conflicts when we add integration tests (same reason).

```suggestion
  name                = "${terraform.workspace}-they-test-powershell"
```

---

In my mind, this should be done once for all examples (probably during https://github.com/THEY-Consulting/they-terraform/issues/31)